README.md

```diff
@@ -1,2 +1,109 @@
+
+We discussed other options with the CGA team that would involve them making non-neglible changes to the exising legacy codebase:
+
+- Experimenting with [Lift’s REST helpers][lift-rest]
+- Building a bespoke [JSON:API] service
+
+After evaluating the tradeoffs, both teams decided that the solution we chose would be best--it ensures we have access to the data Bamboo needs and requires no modification of the legacy app.
+
+### Design components
+
+#### The problem
+
+Some of the content included in the [prototype designs] are not available in the CGA application. This includes:
+
+- Issue guide cover images
+- Issue guide categories
+
+#### The solution
+
+After consulting with our senior designer, we feel like we can build Bamboo without this content and not unduly compromise the user experience.
+
+This content, though not essential, would be nice to have and we’ve identified a couple of potential pathways for getting access to it:
+
+- The CGA team could add fields to the legacy app to allow issue guide authors to specify an image and category
+- The [EVE website][eve], a WordPress app, has the content we need and there are plugins for WordPress that can provide a drop-in web API. The difficult part of this approach, however, is linking up the WordPress data with the correct CGA issue guides.
+
+As both of these options require work from the CGA team, and not much work on our part, we feel that ...
+
+### Out-of-band decisions
+
+#### The problem
+
+In the current CGA app, participant decisions, e.g. chosen options, actions, evaluations, are recorded as part of an event stream during the forum experience. Bamboo records these decisions outside of the forum experience and CGA has not yet decided how they will be reconciled with the the event stream paradigm.
+
+### The solution
+
+NewAperio will store user decisions as part of the Bamboo application. When a complete set of decisions has been recorded, Bamboo will send a webhook event to CGA with information about how to retrieve the records. This decouples Bamboo’s implementation from the CGA app, leaving the CGA team to design their solution without restrictions.
+
+### Participant accounts
+
+#### The problem
+
+Persistent user accounts are not part of the existing CGA application, that is to say, a new user is generated per participant, per forum and does not exist out of that context. CGA is working on implementing peristent accounts but there is no planned solution for ingesting external account data.
+
+#### The solution
+
+Bamboo will implement its own user authentication system, which will allow users to sign up for multiple forums without having to log in each time. When webhook events are sent to CGA, indentifying user information will be included. Like the decisions themselves, this somewhat decouple the two apps and allows CGA to build a solution without inteference or restriction.
```

Comment:
```suggestion
Bamboo will implement its own user authentication system, which will allow users to sign up for multiple forums without having to log in each time. When webhook events are sent to CGA, identifying user information will be included. Like the decisions themselves, this will somewhat decouple the two apps and will allow CGA to build a solution without interference or restriction.
```